src/main/java/service/operation/BalanceOperation.java

```diff
@@ -0,0 +1,17 @@
+package service.operation;
+
+import dao.FruitDao;
+import dao.FruitDaoImpl;
+
+public class BalanceOperation implements OperationHandler {
+    private FruitDao fruitDao;
+
+    public BalanceOperation() {
+        this.fruitDao = new FruitDaoImpl();
+    }
```

Comment:
Remove the empty line at the start of the constructor implementation to follow the common coding conventions mentioned in the checklist.